Java/samples/simple-console/src/main/java/com/microsoft/recognizers/text/samples/simpleconsole/Sample.java

```diff
@@ -94,7 +96,35 @@ public static void main(String[] args) throws Exception {
 
                 // Datetime recognizer This model will find any Date even if its write in colloquial language
                 // E.g "I'll go back 8pm today" will return "2017-10-04 20:00:00"
-                DateTimeRecognizer.recognizeDateTime(query, culture)
+                DateTimeRecognizer.recognizeDateTime(query, culture),
+
+                // PhoneNumber recognizer will find any phone number presented
+                // E.g "My phone number is ( 19 ) 38294427."
+                SequenceRecognizer.recognizePhoneNumber(query, culture, SequenceOptions.None, true),
```

Comment:
Similarly to the other models, the last two parameters should not be required, unless a user wants to override the defaults.

Reply:
Done!